Source/ACE.Entity/Enum/WeenieType.cs

```diff
@@ -75,6 +75,7 @@ public enum WeenieType : uint
         SocialManager,
         Pet,
         PetDevice,
-        CombatPet
+        CombatPet,
+        Foci
```

Comment:
Foci did not have their own WeenieType. They were assigned to WeenieType.Generic